packages/workspace/src/ProjectManager/RemoteProjectManager.js

```diff
@@ -16,6 +17,19 @@ export default class RemoteProjectManager extends BaseProjectManager {
 
     this.remote = true
     this.prefix = 'private'
+    const projectOwner = this.projectRoot.split("/")[0]
+    this.projectOwner = projectOwner
+    this.userOwnProject = Auth.profile.username === projectOwner
+  }
+
+  togglePublic = async (aim = void 0) => {
+    if (aim === 'public') this.prefix = 'public'
```

Comment:
https://github.com/ObsidianLabs/electron-components/pull/111/files#diff-ad9dfd902c7c6f996d2262d5c1ffe6193c6a818b9a68c735595f4aa420983837R81, from the context, it seems that the aim parameter has only two values, ' public ' & ' private '
```suggestion
     this.prefix = aim
```